packages/nitro-protocol/src/index.ts

```diff
@@ -48,4 +55,10 @@ export {
   getDepositedEvent,
   isAllocationOutcome,
   isGuaranteeOutcome,
+  VariablePart,
+  // test helperts -- TODO move these to devtools package
```

Comment:
Should this be `test helpers`?

Reply:
mos def. Hopefully I will get those functions moved and the comment wiped before this PR is ready :-)